serverlessgo/sim/integration_tests/test.go

```diff
@@ -17,17 +17,41 @@ func (rs *collectorListener) RequestFinished(req *sim.Request) {
 }
 
 // TODO(david): Improve integration tests
+// - read from a file the simulation input/output.
 func main() {
-	duration := 10 * time.Millisecond
-	idlenessDeadline := 10 * time.Millisecond
-	input := [][]sim.InputEntry{{{200, 10}}}
+	duration := 80 * time.Millisecond
+	idlenessDeadline := 6 * time.Millisecond
+	input := [][]sim.InputEntry{
+		{{200, 0.015}, {200, 0.008}, {503, 0.0001}},
+		{{200, 0.011}},
+		{{200, 0.005}, {200, 0.005}, {503, 0.0002}},
+	}
 	optimized := false
 	var collector collectorListener
-	sim.Run(duration, idlenessDeadline, sim.NewPoissonInterArrival(1), input, &collector, optimized)
-	got := collector[0]
-	want := sim.Request{ID: 0, Status: 200, ResponseTime: 10, Hops: []int{0}}
-	if !reflect.DeepEqual(got, want) {
-		log.Fatalf("want:%+v got:%+v", got, want)
+	res := sim.Run(duration, idlenessDeadline, sim.NewConstantInterArrival(0.01), input, &collector, optimized)
+	
+	instancesWant, instancesGot := 5, len(res.Instances)
```

Comment:
Lembrando que go prefere variáveis com nomes pequenos, principalmente quando usadas num escopo tão curto. Mais sobre isso, [aqui](https://github.com/golang/go/wiki/CodeReviewComments#variable-names). Também gostaria de lembrar que não precisa de `reflect.DeepEqual` para comparar dois números.

```go
if len(res.Instances) != 5 {
    log.Fatalf("number of instances - want:5 got:%+v", len(res.Instances))
}
```

Reply:
Done

Reply:
Not done.

Reply:
?? Não entendi...